pom.xml

```diff
@@ -509,6 +510,16 @@
         <artifactId>httpcore</artifactId>
         <version>4.4.6</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.parquet</groupId>
+        <artifactId>parquet-avro</artifactId>
+        <version>${parquet.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.parquet</groupId>
+        <artifactId>parquet-cli</artifactId>
```

Comment:
is using the "cli" artifact, correct?

Reply:
CsvReader depends on a utility class from cli.
In the future, if we implement our own CSV reader, we can get rid of this dependency.